src/styles/app/30-markdown-editor-pane.css

```
@@ -1023,12 +1023,28 @@
 	padding: 18px var(--editor-readable-content-gutter) 50vh;
 }
 
+:root[data-editor-width-mode="wide"]
+	.markdownEditorPane
+	.tiptapHostInline:not(.is-preview)
+	.tiptapContentInline {
+	padding-right: calc(
+		var(--editor-readable-content-gutter-compact) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)